run.sh

```diff
@@ -1,7 +1,7 @@
 #/bin/bash -e
 
-HOSTNAME=$(curl http://169.254.169.254/latest/meta-data/hostname)
-AVAILABILITY_ZONE=$(curl http://169.254.169.254/latest/meta-data/placement/availability-zone)
+HOSTNAME=$(curl --connect-timeout 5 http://169.254.169.254/latest/meta-data/hostname || echo "localhost")
+AVAILABILITY_ZONE=$(curl --connect-timeout 5 http://169.254.169.254/latest/meta-data/placement/availability-zone || echo "")
 AWS_REGION=${AVAILABILITY_ZONE:0:${#AVAILABILITY_ZONE} - 1}
```

Comment:
This line on will produce: `bash: ${#AVAILABILITY_ZONE} - 1: substring expression < 0` if you start appliance locally.